src/Tests/Tests/XPack/Security/ApiKey/SecurityApiKeyTests.cs

```diff
@@ -120,6 +121,25 @@ public SecurityApiKeyTests(XPackCluster cluster, EndpointUsage usage) : base(new
 						(v, c, r) => c.Security.CreateApiKeyAsync(r)
 					)
 			},
+			{
```

Comment:
This needs to be denoted with a version somehow (maybe an additional parameter taking a version range string?)